### ISSUE TITLE
Can't use install script to install jsx in docker on arch

### ISSUE THREAD

**Issue by LeeSmet** [threefoldtecharchive/jumpscaleX_archived#252]
Running Arch linux, the install command as described in the readme always fails, does not even give the option to install a docker

```
lee@Arch  ~/jsx  curl https://raw.githubusercontent.com/threefoldtech/jumpscaleX/master/install/install.py\?$RANDOM > /tmp/install.py;python3 /tmp/install.py
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 11568  100 11568    0     0  40166      0 --:--:-- --:--:-- --:--:-- 40027
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 64427  100 64427    0     0   311k      0 --:--:-- --:--:-- --:--:--  311k
Traceback (most recent call last):
  File "/tmp/install.py", line 16, in <module>
    IT = spec.loader.load_module()
  File "<frozen importlib._bootstrap_external>", line 407, in _check_name_wrapper
  File "<frozen importlib._bootstrap_external>", line 907, in load_module
  File "<frozen importlib._bootstrap_external>", line 732, in load_module
  File "<frozen importlib._bootstrap>", line 265, in _load_module_shim
  File "<frozen importlib._bootstrap>", line 696, in _load
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/tmp/InstallTools.py", line 1940, in <module>
    MyEnv._init()
  File "/tmp/InstallTools.py", line 1534, in _init
    MyEnv.check_platform()
  File "/tmp/InstallTools.py", line 1486, in check_platform
    UbuntuInstall.ensure_version()
  File "/tmp/InstallTools.py", line 1249, in ensure_version
    raise RuntimeError("Your operating system is not supported")
RuntimeError: Your operating system is not supported
```

**Comment by despiegk** [comment]
am sure install script can be modified to do it

**Comment by LeeSmet** [comment]
Closed in efdc70aa9c74bb4f1531a2fc252118995d69b58a
